devnet/docker-compose.yml

```
@@ -0,0 +1,65 @@
+services: 
+  alice:
+    build:
+      context: ./..
+      dockerfile: devnet/alice.Dockerfile
+      args:
+        DD_API_KEY: ${DD_API_KEY}
+        pubip: 10.5.0.2
+    ports:
+      - "7001"
+      - "8545"
+      - "8546"
+      - "8540"
+      - "9876"
```

nit no need for double quotes :wink: 

```suggestion
      - 7001
      - 8545
      - 8546
      - 8540
      - 9876
```